src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -97,13 +97,15 @@ function BoxAISidebarContent(props: ApiWrapperProps) {
     const renderActions = () => (
         <>
             { renderBoxAISidebarTitle() }
-            { isResetChatEnabled && 
-                <IconButton
-                    aria-label={formatMessage(sidebarMessages.boxAISidebarClear)}
-                    icon={Trash}
-                    onClick={onClearAction}
-                    size="x-small"
-                />
+            { isResetChatEnabled &&
```

Comment:
🔧 I suggest that instead of duplicating this component, please abstract the implementation from the shared feature box ai content answers shared feature, and pass the additional props and expose it through the main index file.

That way, if we want in the future to modify the icon, text or tooltip we have only 1 source of truth for them